subsys/bluetooth/controller/hci_internal.c

```diff
@@ -762,6 +764,7 @@ void hci_internal_le_supported_features(
 
 #ifdef CONFIG_BT_CTLR_ADV_EXT
 	features->params.le_extended_advertising = 1;
+	features->params.advertising_coding_selection = 1;
```

Comment:
Is the feature bit set no matter if coded phy is enabled?

Reply:
We support the API independent of the support of coded phy. If coded PHY is not supported will will just not transmit on the particular coding scheme, and also not raise reports with Coded PHY

Reply:
Ok, then we can keep it as is